tdms/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 # General setup ---------------------------------------------------------------
 cmake_minimum_required(VERSION 3.21)
 
-project(tdms LANGUAGES CXX)
+project(tdms LANGUAGES CXX C)
```

Comment:
I'm not 100% we actually _need_ this.
IIRC it's just so that the HDF5 CMake infrastructure can compile a test program.

- [ ] Try without?